tools/tree-viz.html

```diff
@@ -8,7 +8,7 @@
     <script type="importmap">
       {
         "imports": {
-          "@grafana/lezer-logql": "https://cdn.jsdelivr.net/npm/@grafana/lezer-logql@0.0.18/index.es.js",
+          "@grafana/lezer-logql": "https://cdn.jsdelivr.net/npm/@grafana/lezer-logql@0.1.1/index.es.js",
```

Comment:
Thanks for fixing this 🤩

I wondered if we should just use `@latest` here?

Reply:
Yeah, good point. Updated in https://github.com/grafana/lezer-logql/pull/21/commits/82ae83ce151d3fc156596e99b21ec10e74bb38f0.